src/hmr/hotModuleReplacement.js

```diff
@@ -227,3 +231,8 @@ module.exports = function(moduleId, options) {
 
   return debounce(update, 50);
 };
+
+// Used from tests
+module.exports.clearCache = function() {
+  srcByModuleId = Object.create(null);
+};
```

Comment:
We should use mock for test it, no need it in runtime code

Reply:
How to mock variable inside module?

Currently value inside `srcByModuleId = Object.create(null)` persists between tests and my situation happens only if it's clear.